public_html/style.css

```diff
@@ -494,55 +575,54 @@ img {
 
 .cbailylaw__testimonial-grids {
     display: grid;
-    grid-template-columns: auto auto;
+    grid-template-columns: 50% 50%;
     gap: 50px;
 }
 
-.cbailylaw__testimonial-grid-full p,
 .cbailylaw__testimonial-grid p {
     font: 300 20px/34px var(--font-body);
     color: var(--color-black);
-    min-height: 170px;
     letter-spacing: 0.8px;
-    /* text-transform: capitalize; */
 }
 
 .cbailylaw__testimonial-grid.attribution {
     text-align: right;
 }
 
 /* this style is a problem! Don't use span here*/
-.cbailylaw__testimonial-grid-full a,
-.cbailylaw__testimonial-grid span {
+/* 
+Author: Ratul
+Description: changed the SPAN to P tag added the class name for styling
+*/
+.cbailylaw__testimonial-grid .testimonial__author a,
+.cbailylaw__testimonial-grid .testimonial__author {
     display: inline-block;
     color: var(--color-black);
-    font: 600 20px/30px var(--font-body);
+    font: 600 18px/1 var(--font-body);
     line-height: normal;
     letter-spacing: 0.8px;
-    text-decoration-line: underline;
+    border-bottom: 1px solid var(--color-black-gray);
```

Comment:
Please remove this border-bottom, because it makes the text look like a clickable hyperlink.

Also, please add:

```
text-align: right;
```

Or whatever is needed to align these lines to the right.